.github/workflows/ci-uv.yml

```
@@ -0,0 +1,74 @@
+name: Tests (uv)
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  test-uv:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [windows-latest, ubuntu-latest, macos-latest]
+        python-version: ['3.13']
+
+    steps:
+      - uses: actions/checkout@v5
+
+      # recommended way to install uv, python, (and I guess also cookiecutter) is:
+      #- name: Install the latest version of uv
+      #  uses: astral-sh/setup-uv@v6
+      #- name: Set up Python
+      #  run: uv python install ${{ matrix.python-version }}
+      #
+      #- name: Install cookiecutter and pytest dependencies
+      #  run: uvx install cookiecutter pytest
+
+      - uses: actions/setup-python@v6
+        with:
+          python-version: ${{ matrix.python-version }}
+          cache: pip
+
+      - name: Install the latest version of uv
+        uses: astral-sh/setup-uv@v6
```

Let's set these values here

```suggestion
        uses: astral-sh/setup-uv@v6
          enable-cache: true
          python-version: ${{ matrix.python-version }}
```

---

added these suggested changes